### DISCUSSION TITLE
I have 20-15 (with glasses) vision, and I had to get closer to the screen to see things

### DISCUSSION THREAD

**Discussion by pbiggar** [darklang/dark#4298] [Feedback]
A user says:
>My thoughts, in no particular order:
>- The editor feels more than a little clunky. I sometimes felt like I couldn't type when I thought I should be able to type, delete when I should have been able to, etc. Not unusual for alpha/beta-level software.
>- The text, by default, is super small. I have 20-15 (with glasses) vision, and I had to get closer to the screen to see things. Granted, I'm on a smallish laptop screen, but still...
>- Dark is a cool idea, but almost doesn't seem to go far enough. I could feel the underlying technologies and patterns peeking out at me. Almost like ... what am I gaining here? A somewhat advanced IDE for tech I already have? Or is this really a new paradigm?
>
>I think I hit a bug and ended up closing the tab and not going back. Still interested in following along as you progress.

**Comment by jamonholmgren** [comment]
:wave: This was me.
